apps/vth-frontend/src/app/[locale]/page.tsx

```diff
@@ -23,19 +27,18 @@ export async function generateMetadata({ params: { locale } }: Params): Promise<
 }
 
 const Home = async ({ params: { locale } }: { params: any }) => {
-  const { t } = await useTranslation(locale, ['home-page', 'common']);
+  const { data } = await fetchData({
+    url: createStrapiURL(),
+    query: GET_HOME_PAGE,
+    variables: { locale: locale },
+  });
 
-  // todo: fetch theme data to display homepage cards
+  const { Title, Content } = data.homepage.data.attributes;
```

Comment:
I think we need some validation in case the Title or the content is not provided.
Maybe  something like this 

```jsx

   {data.homepage.data?.attributes?.Title && <Heading1>{data.homepage.data.attributes.Titel}</Heading1>}
   {data.homepage.data?.attributes?.Inhoud && <Markdown>{data.homepage.data?.attributes?.Inhoud}</Markdown>}
      ```